src/parsers.ts

```diff
@@ -1,30 +1,35 @@
 import { RenderFunction, StatelessComponentConfig, StyleFunction } from './model';
+import { isString, isUndefined } from './type-guards';
 
 /* SELECTOR */
 function parseSelector(configOrSelector, config) {
   const selector = config ? config.selector : (configOrSelector as string);
-  if (typeof selector === 'undefined') {
+  if (isUndefined(selector)) {
     throw Error('You should specify a selector.');
   }
   return selector;
 }
 /* TEMPLATE */
 function parseTemplate(renderFnOrTemplate, config) {
   const template = renderFnOrTemplate
-    ? typeof renderFnOrTemplate === 'string'
+    ? isString(renderFnOrTemplate)
       ? renderFnOrTemplate
       : renderFnOrTemplate()
     : config.template;
 
-  if (typeof template === 'undefined') {
+  if (isUndefined(template)) {
     throw Error('You should specify a template.');
   }
   return template;
 }
 
 /* STYLE */
 function parseStyle(styleFnOrStyle, config) {
-  return styleFnOrStyle ? (typeof styleFnOrStyle === 'string' ? [styleFnOrStyle] : [styleFnOrStyle()]) : [config.style];
+  return styleFnOrStyle
```

Comment:
maybe you can add a type guard isDefined for this check?

Reply:
Added.